packages/core/src/awsService/cloudWatchLogs/activation.ts

```diff
@@ -89,6 +90,14 @@ export async function activate(context: vscode.ExtensionContext, configuration:
 
         Commands.register('aws.cwl.changeFilterPattern', async () => changeLogSearchParams(registry, 'filterPattern')),
 
-        Commands.register('aws.cwl.changeTimeFilter', async () => changeLogSearchParams(registry, 'timeFilter'))
+        Commands.register('aws.cwl.changeTimeFilter', async () => changeLogSearchParams(registry, 'timeFilter')),
+    
```

Comment:
looks like theres a couple linting issues. From the root I think you should be able to run `npm run lintfix`